### PR TITLE
📖 Describe clusteradm in packaging.md

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates packaging.md to describe how the clusteradm container image is built.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @pdettori 
/cc @francostellari 
@ezrasilvera 
@nirrozenbaum

File: docs/content/v0.20/packaging.md [hunk 1]
```diff
@@ -211,11 +216,24 @@ This chart creates (among other things) a `Deployment` object that runs a contai
...
+### clusteradm container image
+
+The kubestellar GitHub repository has a script,
+`hack/build-clusteradm-image.sh`, that creates and publishes a
+container image holding the `clusteradm` command from OCM. The source
+of the container image is read from the latest release of
+[github.com/open-cluster-management-io/clusteradm](https://github.com/open-cluster-management-io/clusteradm),
+unless a command line flag says to use a specific version. This script
+also pushes the built container image to
+[quay.io/kubestellar/clusteradm](https://quay.io/repository/kubestellar/clusteradm)
```
[the remark this answers is not in the document]
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:228]: No, we did not. We do not.

[the remark this answers is not in the document]
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:228]: See https://github.com/kubestellar/kubestellar/blob/main/docs/content/v0.20/packaging.md for an in-progress doc on the current truth. See a recent Slack conversation that confirmed that we are OK with using a mixture of quay and ghcr.

**Comment by nirrozenbaum** [comment]
I have no other comments/questions from my side.
/lgtm

File: docs/content/v0.20/packaging.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pdettori) [reply docs/content/v0.20/packaging.md:228]: The current thinking is to use quay.io for OCI images built from other open source projects (when such projects do not publish OCI images themselves for some binaries we need to run in containers, as it is in the case of clusteradm and helm), and to use ghcr.io for publishing OCI images for source code maintained by our KubeStellar community.

File: docs/content/v0.20/packaging.md [hunk 2]
```diff
@@ -211,11 +216,24 @@ This chart creates (among other things) a `Deployment` object that runs a contai
...
+using a tag that equals the ocm/clusteradm version that the image was
+built from.
+
 ### KubeFlex PostCreateHooks
 
 There are two `PostCreateHook` objects defined in [config/postcreate-hooks](../../../config/postcreate-hooks). What uses these copies of those hook definitions?
 
-- `ocm.yaml` adds `clusteradm` by running a container using the image `quay.io/kubestellar/clusteradm:0.7.2` which is built from [the OCM source](https://github.com/open-cluster-management-io/clusteradm) using the script [build-clusteradm-image.sh](../../../hack/build-clusteradm-image.sh).
+- `ocm.yaml` adds `clusteradm` by running a container using the image `quay.io/kubestellar/clusteradm:0.7.2`; see [above](#clusteradm-container-image) about the source of that.
 - `kubestellar.yaml` runs container image `quay.io/kubestellar/helm:v3.14.0` (which is built from [the Helm source](https://github.com/helm/helm/tree/v3.14.0) by a process that we need to document) to instantiate the chart from `oci://ghcr.io/kubestellar/kubestellar/kubestellar-operator-chart` with chart version `0.20.0-alpha.1`, which is built by (what?) from (what? probably answered above).
```
Comment (pdettori) [inline docs/content/v0.20/packaging.md:237]: As for the building helm image, isn't that done in the script https://github.com/kubestellar/kubestellar/blob/main/hack/build-helm-image.sh ?

File: docs/content/v0.20/packaging.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nirrozenbaum) [reply docs/content/v0.20/packaging.md:228]: got it, thanks.

File: docs/content/v0.20/packaging.md (hunk 2 above)
Comment (MikeSpreitzer) [reply docs/content/v0.20/packaging.md:237]: Thanks. I will update that in a future PR.

**Pull request merged**